packages/graphql/lib/schema-builder/storages/type-metadata.storage.model.ts

```diff
@@ -0,0 +1,221 @@
+import {
+  ClassDirectiveMetadata,
+  ClassExtensionsMetadata,
+  ClassMetadata,
+  MethodArgsMetadata,
+  PropertyDirectiveMetadata,
+  PropertyExtensionsMetadata,
+  PropertyMetadata,
+  ResolverClassMetadata,
+} from '../metadata';
+import { ObjectTypeMetadata } from '../metadata/object-type.metadata';
+
+export class MapByName<T> {
```

Comment:
Can we create a dedicated folder, let's say, "collections" and then have a file per class there?

Reply:
Sure, that's a good idea. I didn't know where to place this file but your suggestion is appropriate

Reply:
Done.